modules/logging-in-by-using-the-web-console.adoc

```diff
@@ -10,20 +12,20 @@ The `kubeadmin` user exists by default after an {product-title} installation. Yo
 .Prerequisites
 
 * You have access to the installation host.
-* You have completed a cluster installation and all cluster Operators are available.
```

Comment:
Interesting, you must have been following a plan 😁

Reply:
Yes, I removed the word "have" from all of the prerequisites in these assemblies, as suggested in [this comment from the earlier peer review](https://github.com/openshift/openshift-docs/pull/28079#discussion_r543705148). :-)